docs/other.rst

```diff
@@ -141,3 +141,38 @@ Lego Port
 
 .. autoclass:: ev3dev2.port.LegoPort
     :members:
+
+Wheels
```

Comment:
In all honesty, I think the `other` docs page is pretty unweildy. We have so much stuff here that it just becomes difficult to browse. What do you think about breaking from tradition here and giving this its own file, and then we can track an issue for splitting out some of the other classes (like buttons) which are getting pretty big?

Reply:
Let's commit the wheels docs here for now and then we can figure out how we want to split this file up afterwards?

Reply:
Filed #559 to track this part